libs/sdk-ui-dashboard/src/presentation/widget/insight/ViewModeDashboardInsight/Insight/useDashboardInsightDrills.ts

```diff
@@ -57,7 +58,9 @@ export const useDashboardInsightDrills = ({
     );
 
     const isInEditMode = useDashboardSelector(selectIsInEditMode);
-    const rawDrillableItems = useDashboardSelector(selectDrillableItemsByWidgetRef(widget.ref));
+    const rawDrillableItems = useDashboardSelector(
+        selectDrillableItemsByWidgetRef(widget.ref, disableDrillDownOnWidget),
```

Comment:
Does this selector signature need to be extended? You already have `widget.ref`. Can't you use it inside of the selector to get `insight` and inside of this selector get the prop? You would just change the selector and do not need to update all the other places where it is called.

Reply:
I have updated this. Thank you!